modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookManagementToolbarDisplayContext.java

```diff
@@ -126,6 +144,46 @@ public Map<String, Object> getAdditionalProps() {
 
 				return exportStyleBookEntriesURL.toString();
 			}
+		).put(
+			"frontendTokenDefinitionProviders",
+			() -> {
+				long companyId = _themeDisplay.getCompanyId();
+
+				List<Map<String, Object>> frontendTokenDefinitionProviders =
+					new ArrayList<>();
+
+				for (FrontendTokenDefinition frontendTokenDefinition :
+						_frontendTokenDefinitionRegistry.
+							getFrontendTokenDefinitions(companyId)) {
+
+					String themeId = frontendTokenDefinition.getThemeId();
+
+					String name = themeId;
+
+					Theme theme = ThemeLocalServiceUtil.fetchTheme(
+						companyId, themeId);
+
+					if (theme != null) {
+						name = theme.getName();
+					}
+					else {
+						CET cet = _cetManager.getCET(companyId, themeId);
+
+						if (cet != null) {
+							name = cet.getName();
+						}
+					}
+
+					frontendTokenDefinitionProviders.add(
+						HashMapBuilder.<String, Object>put(
+							"name", name
```

Comment:
The name shouldn't just be the theme or CX name. It needs to be suffixed by it's type. So it should be either `x-theme` or `x-theme-css-client-extension` depending on what it is. This is important because we could have a theme and a theme css client extension with the same name.

Reply:
Just to confirm, should these language keys be the following:
x-theme={0} Theme
x-theme-css-client-extension={0} Theme CSS Client Extension

I referred to this https://github.com/brianchandotcom/liferay-portal/pull/58629#issuecomment-390004381 and labeled them as titles